sktime/classification/shapelet_based/tests/test_rocket_classifier.py

```diff
@@ -41,6 +41,27 @@ def test_rocket_ensemble_on_gunpoint():
     testing.assert_array_equal(probas, rocket_e_gunpoint_probas)
 
 
+def test_rocket_parallel_ensemble_on_gunpoint():
```

Comment:
This seems to be the same test logic as the above test, could we use `pytest.mark.parametrize("n_jobs", [1, 8])` to iterate over it rather than duplicating the code?